backends/p4tools/modules/testgen/targets/bmv2/target.cpp

```diff
@@ -77,7 +77,7 @@ const BMv2_V1ModelProgramInfo *BMv2_V1ModelTestgenTarget::initProgram_impl(
 
 Bmv2TestBackend *BMv2_V1ModelTestgenTarget::getTestBackend_impl(
     const ProgramInfo &programInfo, SymbolicExecutor &symbex,
-    const boost::filesystem::path &testPath, boost::optional<uint32_t> seed) const {
+    const std::filesystem::path &testPath, boost::optional<uint32_t> seed) const {
```

Comment:
Can we change `boost::optional` to `std::optional` too?

Reply:
Yes, the corresponding PR can be found here: https://github.com/p4lang/p4c/pull/3942

Reply:
I will also try to do this for std::variant, but that is a bit more tricky.

Reply:
got it, is there a reason we are moving away from boost - is it to avoid extra dependencies?

Reply:
Yes, boost adds compile time and binary size overhead. It is also one of the major sources for build issues (next to protobuf). So I am trying to at least fix some low-hanging fruits.